drafts/prometheus-alertmanager-rhocp-open-liberty.adoc

```diff
@@ -0,0 +1,207 @@
++
+```
+spec:
+  groups:
+    - name: ./example.rules
+      rules:
+        - alert: ExampleAlert
+          expr: vector(1)
+    - name: libertyexample
+      rules:
+      - alert: heapUsageTooHigh
+        expr: base_memory_usedHeap_bytes / base_memory_maxHeap_bytes > 0.9
+        for: 1m
+        labels:
+          severity: warning
+        annotations:
+          summary: "Heap usage is too high"
+          description: "{{ $labels.instance }} heap usage is too high"
+```
++
+This rule will send an alert called `heapUsageTooHigh` when the PromQL query `base_memory_usedHeap_bytes / base_memory_maxHeap_bytes` is greater than `0.9` (90%) for one minute. (For testing purposes, feel free to change the threshold to something smaller, like `0.05`.)
++
+If you're new to PromQL, check out link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Querying Prometheus].
++
+Note that there's a rule group with the name `./example.rules` created by default. This is a rule that always fires an alert, so you can keep this for now to test your configuration.
++
+. Change the labels to match the `ruleSelector` in your Prometheus. For example, if you used the `prometheus.yaml` in the Kabanero guide, your Prometheus YAML would contain:
++
+```
+ruleSelector:
+    matchLabels:
+      prometheus: k8s
+      role: prometheus-rulefiles
+```
++
+So your PrometheusRule labels must also match:
++
+```
+labels:
+    prometheus: k8s
+    role: prometheus-rulefiles
+```
++
+. Navigate to your Prometheus route and click on Status > Rules to verify that they've been loaded in properly.
++
+image::/img/blog/prometheusAM_rhocp_promUI_rules.png[Prometheus Rules on UI, align="left"]
++
+. To check that your alerts are firing, click on Alerts. Since we did not remove the `./example.rules` rule group, you should already see at least one alert firing. To ensure that your custom Liberty rules are also working, generate some data or drastically lower the threshold of the PromQL query.
++
+image::/img/blog/prometheusAM_rhocp_promUI_alerts.png[Prometheus Alerts on UI, align="left"]
+
+== Create and configure Prometheus Alertmanager to pass alerts to Slack
+
+In this section, you'll create and configure the Prometheus Alertmanager to send customized messages to your Slack channel based on the alerting rules we set up earlier.
+
+=== Creating and configuring the Prometheus Alertmanager to communicate with Slack
+
+Now that Prometheus is set up with rules for our Liberty server, we can create a Prometheus Alertmanager instance with the ability to connect to Slack.
+
+. Create a file named `alertmanager.yaml` (do not change the name) - take note of the directory, as we will be creating a secret using this file. Note that this file is configured the same way as we did in the previous blog post. We'll start with a simple configuration. Your file will look like this:
```

Comment:
Update
"Create a file named `alertmanager.yaml` (do not change the name) - take note of the directory, as we will be creating a secret using this file. Note that this file is configured the same way as we did in the previous blog post."
To
"Create a file named `alertmanager.yaml`, and remember the directory, as we will be creating a secret using this file. This file is configured using the same steps from the original blog post."